modules/imgproc/test/test_thresh.cpp

```diff
@@ -57,16 +57,22 @@ class CV_ThreshTest : public cvtest::ArrayTest
     int thresh_type;
     double thresh_val;
     double max_val;
+    int extra_type;
 };
 
 
-CV_ThreshTest::CV_ThreshTest()
+CV_ThreshTest::CV_ThreshTest(int test_type)
 {
+    assert( (test_type & CV_THRESH_MASK) == 0 );
```

Comment:
"C" asserts are banned in OpenCV including test code.
Use `CV_Assert()` instead.